src/apps/loan-list/materials/selectable-material/selectable-material.tsx

```diff
@@ -15,8 +14,8 @@ import StatusMessage from "./StatusMessage";
 interface SelectableMaterialProps {
   loan: LoanType;
   disabled?: boolean;
-  materialsToRenew: FaustId[];
-  onChecked?: (faust: FaustId) => void;
+  materialsToRenew: number[];
+  onChecked?: (loanId: number) => void;
```

Comment:
I think we could put the loan id along the other knowns id's by creating a `LoanId` type in /core/utils/types/ids. It is already used in multiple places and we can benefit from it when the functionality with loan id expands.

Reply:
export type LoanId = number;  ?

Reply:
Yes same as FaustId:
```
export type FaustId = `${number}`;